src/celastro/date.h

```diff
@@ -84,10 +90,30 @@ class Date
     int minute{ 0 };
     int wday{ 0 };               // week day, 0 Sunday to 6 Saturday
     int utc_offset{ 0 };         // offset from utc in seconds
-    std::string tzname{ "UTC" }; // timezone name
+    char tzname[8]{ 'U', 'T', 'C', '\0' }; // timezone name
     double seconds{ 0.0 };
 };
 
+inline Date operator-(const Date &a, const Date &b)
+{
+    return Date(static_cast<double>(a) - static_cast<double>(b));
+}
+
+inline Date operator+(const Date &a, const Date &b)
```

Comment:
I'm not sure about this operator - probably would make more sense if the right-hand side were a time interval type rather than a date.